vault/activity_log_testing_util.go

```diff
@@ -0,0 +1,167 @@
+package vault
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hashicorp/vault/sdk/logical"
+	"github.com/hashicorp/vault/vault/activity"
+)
+
+// Return the in-memory activeEntities from an activity log
+func (c *Core) GetActiveEntities() map[string]struct{} {
+	out := make(map[string]struct{})
+
+	c.stateLock.RLock()
+	c.activityLog.fragmentLock.RLock()
+	for k, v := range c.activityLog.activeEntities {
+		out[k] = v
+	}
+	c.activityLog.fragmentLock.RUnlock()
+	c.stateLock.RUnlock()
+
+	return out
+}
+
+// GetCurrentEntities returns the current entity activity log
+func (a *ActivityLog) GetCurrentEntities() *activity.EntityActivityLog {
+	a.l.RLock()
+	defer a.l.RUnlock()
+	return a.currentSegment.currentEntities
+}
+
+// WriteToStorage is used to put entity data in storage
+// `path` should be the complete path (not relative to the view)
+func WriteToStorage(t *testing.T, c *Core, path string, data []byte) {
```

Comment:
I think this should be moved to the testhelpers package.

Reply:
`testhelpers` can't be imported into the `vault` package (creates an import cycle), and `WriteToStorage` is being used in the `vault` package. Is there somewhere else where this could go